### PR TITLE
Feat/condo/doma 4429/new schema bank transaction

### PR DESCRIPTION
**Pull request by AntonAL**

### CODE REVIEW

File: apps/condo/domains/organization/utils/accessSchema.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ async function checkOrganizationPermission (userId, organizationId, permission)
         organization: { id: organizationId },
     })
 
+    console.debug('>>>>> employeeRole', employeeRole)
```
Comment (VKislov) [inline apps/condo/domains/organization/utils/accessSchema.js:21]: Seems forgotten

File: apps/condo/domains/banking/schema/BankTransaction.test.js [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+
+                await expectToThrowAuthenticationErrorToObjects(async () => {
+                    await BankTransaction.getAll(anonymous, { id: objCreated.id }, { sortBy: ['updatedAt_DESC'] })
+                })
+            })
+        })
+    })
+
+    describe('Validation tests', () => {
+        test('Should have correct dv field (=== 1)', async () => {
```
Comment (VKislov) [inline apps/condo/domains/banking/schema/BankTransaction.test.js:509]: It seems to be redundant. The `DvAndSender` plugin has a validator for this. If you try to create in a test with a different `Dv` value, an error will be thrown. You can see here `packages/keystone/plugins/dvAndSender.js:128`

File: apps/condo/domains/banking/schema/BankTransaction.test.js [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+                })
+                const [contractorAccount] = await createTestBankContractorAccount(support, organization)
+
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await createTestBankTransaction(support, account, contractorAccount, integrationContext, organization)
+                })
+            })
+
+            test('user can if it is an employee of organization with "canManageBankTransactions" permission', async () => {
+                const userClient = await makeClientWithNewRegisteredAndLoggedInUser()
```
Comment (VKislov) [inline apps/condo/domains/banking/schema/BankTransaction.test.js:86]: Perhaps it's better to put it in `beforeAll`. Each test creates a new `organization` and `employee` instance. It seems that in this case there will be no collisions in different tests when using the same user and different organizations

Comment (VKislov) [reply apps/condo/domains/banking/schema/BankTransaction.test.js:86]: Same for tests below

File: apps/condo/domains/banking/access/BankTransaction.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+const { checkPermissionInUserOrganizationOrRelatedOrganization, queryOrganizationEmployeeFor,
+    queryOrganizationEmployeeFromRelatedOrganizationFor,
+} = require('@condo/domains/organization/utils/accessSchema')
+const { getById } = require('@open-condo/keystone/schema')
+
+async function canReadBankTransactions ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin) return {}
```
Comment (sitozzz) [inline apps/condo/domains/banking/access/BankTransaction.js:16]: Why not simply boolean value?

**Review by sitozzz — APPROVED** [review]

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/banking/schema/BankTransaction.test.js (hunk 3 above)
Comment (VKislov) [reply apps/condo/domains/banking/schema/BankTransaction.test.js:91]: Yes, it is better to create an organization and other entities related to it clean. I am looking at userClient. It seems that userClient can be the same in different organizations.

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**